### DISCUSSION TITLE
esp01 tasmota shutter mg90s

### DISCUSSION THREAD

**Discussion by xjustbmw** [arendst/Tasmota#11499] [Q&A] [answered]
hello, i am having a servo MG90S connected to esp01. aparently it only wants to move if i send 10% more increments than the actual position. like a deadband . is this software in tasmota or hardware in motor ? here is my output 20:41:42.695 RSL: stat/tasmota_784807/RESULT = {"Shutter1":{"Position":21,"Direction":0,"Target":23}}
20:41:42.699 RSL: stat/tasmota_784807/RESULT = {"ShutterPosition1":21}
target 23, still stays at 21.. if i send target 30 it moves to the right position

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
